scripts/gen_categories_dataset.py

```diff
@@ -329,10 +349,10 @@ class DataHarvester:
         "categories_tags", "categories_hierarchy", "categories_properties",
         # features
         "nutriments",
-        "ingredients_{LC}", "ingredients_text_{LC}",
-        "product_name_{LC}",
+        "ingredients_tags", "ingredients_original_tags", "ingredients_text_{LC}",
```

Comment:
Note for later: do we need/want to keep 'ingredients_original_tags' in addition to 'ingredients_tags'? I don't remember the difference (sorry I know you explained it already...)